bundles/config/org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,22 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Configuration Discovery Usb-Serial Linux Sysfs Tests
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs.test;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.10.0.qualifier
+Fragment-Host: org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs
+Import-Package:  
+ com.google.common.collect,

```

Comment:
avoid! (see below)

Reply:
I will remove this import and adapt the test code accordingly.